VAMobile/src/screens/HealthScreen/Appointments/AppointmentDetailsCommon/PreferredDateAndTime.tsx

```diff
@@ -17,22 +17,22 @@ const PreferredDateAndTime: FC<PreferredDateAndTimeProps> = ({ attributes }) =>
   const { t } = useTranslation(NAMESPACE.COMMON)
   const theme = useTheme()
 
-  const { proposedTimes } = attributes || ({} as AppointmentAttributes)
+  const { proposedTimes, phoneOnly } = attributes || ({} as AppointmentAttributes)
 
   if (isAppointmentPending && !!proposedTimes?.length) {
     const filteredTimes = proposedTimes?.filter(({ date }) => {
       return !!date
     })
 
     return (
-      <Box mt={theme.dimensions.standardMarginBetween}>
+      <Box mt={phoneOnly ? undefined : theme.dimensions.standardMarginBetween}>
         <TextView variant="MobileBodyBold" accessibilityRole="header">
           {t('appointments.pending.preferredDateAndTime')}
         </TextView>
         {filteredTimes?.map(({ date, time }, index) => {
           const timeSlot = time === AppointmentProposedTimesPeriodConstant.AM ? t('appointments.pending.inMorning') : t('appointments.pending.inAfternoon')
           return (
-            <TextView key={index} variant="MobileBody">
+            <TextView key={index} variant="MobileBody" paragraphSpacing={phoneOnly && index === filteredTimes.length - 1}>
```

Comment:
is this a WCAG requirement? Can you explain the logic to avoid me digging?

Reply:
yes this is a WCAG requirement. The logic is that basically it is a list of preferred times and after a list you are supposed to have paragraph spacing. So I add it to the last textview added in the list.